rpc/jsonrpc/server/ws_handler.go

```diff
@@ -440,7 +440,7 @@ func (wsc *wsConnection) writeRoutine() {
 				continue
 			}
 			if err = wsc.writeMessageWithDeadline(websocket.TextMessage, jsonBytes); err != nil {
-				wsc.Logger.Error("Failed to write response", "err", err, "msg", msg)
+				wsc.Logger.Error("Failed to write response", "err", err)
```

Comment:
we can instead have a second error as Debug containing the message, but I doubt that's useful too as it would just make the debug logs hard to navigate. Maybe we could get clever and only output the first 100bytes and last 100 bytes to have an idea why it's happening. But unless the node operators object to this, or users want to debug something, I prefer the message not to be output to stdout